.github/actions/integration-tests/run-ci-stage1

```diff
@@ -187,12 +187,20 @@ for userenv in ${CI_ACTIVE_USERENVS}; do
     done
 done
 
+run_cmd "crucible es init"
+
+run_cmd "crucible es rebuild"
```

Comment:
It does, sort of.  When we do a run it will automatically start and initialize ES for us as part of the post processing` phase.  However, it can also be invokved explicitly like this so I added it as yet another thing to test since the code paths are not exactly the same.

Reply:
understood, thx for clarifying